rtt/src/lib.rs

```diff
@@ -78,4 +78,8 @@ pub enum Error {
     /// Wraps errors propagated up from probe-rs.
     #[error("Error communicating with probe: {0}")]
     Probe(#[from] probe_rs::Error),
+
+    /// Wraps errors propagated up from scroll.
+    #[error(transparent)]
```

Comment:
Errors should always have a meaningful description such that the user can understand what the error means. So transparent should not be used for user facing errors. For internal ones I think it can be okay :)
Also, see above comment about introducing this variant :)

Reply:
I would like to keep this variant, and have added a more user friendly message. In the `probe-rs-debugger`, I will iterate through the sources of the error, so that the user gets the more friendly message, while the 'root' cause is also captured. I believe this will make it easier for future maintainers to identify the cause of Issues and hopefully fix them. Are you OK with that?

Reply:
Good catch on the user facing message. I chose to introduce a variant here, because it helps to pass specific info about the operation that caused the error.